wordpress-mobile/WordPressMocks
# Add more media for media library screenshots

#### Proposed Changes

Update the `media` endpoint mock to include more media, so the media library looks full in iPad screenshots.

Notice that in the Before screenshot below, in the very last row of the media library images there's a blank spot. This fixes that so the last row is entirely full.

Before|After
-|-
<img width="1366" alt="iPad Pro (12 9-inch) (2nd generation)-6-light-Media" src="https://user-images.githubusercontent.com/8658164/96008773-cec5ed00-0e37-11eb-9c25-46c695c8d44e.png">|<img width="1366" alt="iPad Pro (12 9-inch) (2nd generation)-6-light-Media" src="https://user-images.githubusercontent.com/8658164/96008710-bf46a400-0e37-11eb-8963-1ffdc8ee84d5.png">


#### To Test



* Run the WordPressScreenshotGeneration test on an iPad Pro (12.9-inch, 2nd or 3rd gen) in https://github.com/wordpress-mobile/WordPress-iOS/pull/15099 and confirm the Media section is full of media as in the screenshot above.
